src/delivery-options/components/Pickup/Map/Leaflet.vue

```diff
@@ -337,13 +339,13 @@ export default {
     },
 
     /**
-     * Fit the bounds of the map to the visible markers to determine the center, then apply the set zoom level.
+     * Fit the bounds of the map to the closest few markers so at least something is visible no matter the distance.
      */
     fitToMarkers() {
-      const bounds = this.markers.map((marker) => marker.latLng);
+      const firstFewMarkers = this.markers.slice(0, MINIMUM_VISIBLE_MARKERS);
+      const bounds = firstFewMarkers.map((marker) => marker.latLng);
 
       this.map.fitBounds(bounds);
-      this.map.setZoom(this.zoom);
```

Comment:
Hebben we nog steeds de `this.zoom` nodig in onze data? Ik ga uit van wel, maar wellicht gebruiken we het nu nergens meer.

Reply:
Nee, scherp! verwijderd :)

Reply:
👍🏻